### ISSUE TITLE
Support regression for MultiLabelClassificationModel

### ISSUE THREAD

**Issue by thymian-crypto** [ThilinaRajapakse/simpletransformers#1116]
I would like to use the `MultiLabelClassificationModel` for a multi-task learning problem with continuous targets, i.e. instead of `[0, 1, 1, 0]` having labels in the form of `[0.2, 0.9, 0.4, 0.1]`. 

I was able to train the `MultiLabelClassificationModel` with setting `model_args.regression = True`, but the results are nonsensical as per default, the model minimizes Binary Cross Entropy Loss.

I believe this is related to issue #751, as being able to define a custom loss function for (e.g. in this case, average error or correlation) would likely solve the issue.] 

Thank you for your time and dedication!

**Edit**: Alternatively, any hints how to implement a custom loss function (for an ordinary regression task to which I would fall back to), e.g. MAE or RMSE, would be greatly appreciated.

**Comment by kinoute** [comment]
@thymian-crypto Did you find a solution? I'm working on a multi-label/output regression model and had encountered the same issue.
